frontend/lib/DataModels/EndpointSummary.dart

```diff
@@ -1,20 +1,23 @@
-class EndpointSummary{
+class EndpointSummary {
   final int id;
   final String label;
 
   EndpointSummary(this.id, this.label);
 
-  EndpointSummary.fromJson(Map json):
-    id = json["id"],
-    label = json["label"];
+  EndpointSummary.fromJson(Map json)
+      : id = json["id"],
+        label = json["label"];
+
+  EndpointSummary.empty()
```

Comment:
Jest nam to potrzebne? Bo nie widzę, żeby było używane w projekcie

Reply:
na przyszłość, jak testowałem to sobie to zrobiłem. Można wywalić w sumie